public/app/core/components/RolePicker/RolePickerInput.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { FormEvent, HTMLProps, MutableRefObject, useEffect, useRef } from 'react';
+import { css, cx } from '@emotion/css';
+import { useStyles2, getInputStyles, sharedInputStyle, styleMixins, Tooltip, Icon } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { ValueContainer } from './ValueContainer';
+import { Role } from '../../../types';
+
+const stopPropagation = (event: React.MouseEvent<HTMLDivElement>) => event.stopPropagation();
+
+interface InputProps extends HTMLProps<HTMLInputElement> {
+  appliedRoles: Role[];
+  builtInRole: string;
+  query: string;
+  isFocused?: boolean;
+  disabled?: boolean;
```

Comment:
same comment as about with boolean values.

```suggestion
  isDisabled?: boolean;
```